hearth/lib/hearth/interceptor/context.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Hearth
+  module Interceptor
+    # Context provided to interceptor hooks methods.
```

Comment:
These should have real yardocs eventually

Reply:
Yeah - will spend time on documentation as a finishing touch once other parts are finialized.